include/crypto/ml_dsa.h

```diff
@@ -1,12 +1,3 @@
-/*
- * Copyright 2024 The OpenSSL Project Authors. All Rights Reserved.
- *
- * Licensed under the Apache License 2.0 (the "License").  You may not use
- * this file except in compliance with the License.  You can obtain a copy
- * in the file LICENSE in the source distribution or at
- * https://www.openssl.org/source/license.html
- */
-
```

Comment:
Do we remove the full copyright notice from non-public headers?

Reply:
This seems to be in lots of other files... If its a issue it an issue for another PR.